server/src/db/models/task.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+class Task {
+  id;
+  userId;
```

Comment:
Explicitly defining class fields might be redundant in this case, because constructor function does both - define and assign class instance properties 🤔